datafusion/common/src/scalar/mod.rs

```diff
@@ -3471,9 +3560,6 @@ impl fmt::Debug for ScalarValue {
             ScalarValue::List(_) => write!(f, "List({self})"),
             ScalarValue::LargeList(_) => write!(f, "LargeList({self})"),
             ScalarValue::Struct(struct_arr) => {
-                // ScalarValue Struct should always have a single element
-                assert_eq!(struct_arr.len(), 1);
-
```

Comment:
was this removal intentional?

Reply:
oops, I removed the wrong lines. I'll revert it. Thanks.